content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,305 @@
+`includeAssociatedData` | No        | [`string`]  | When provided, the group export will return or omit a pre-defined set of FHIR resources associated with the request. A string of comma-delimited values. Example: `includeAssociatedData=LatestProvenanceResources`
+`_outputFormat`         | No        | [`string`]  | The format for the requested bulk data files to be generated. Example: `_outputFormat=application/ndjson`
+
+_Notes_
+
+* `_type` valid parameters are `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `Location`, `MedicationRequest`, `Patient`, `Procedure`, `Observation`, `Organization`, `Practitioner`, `Provenance`
+* When the `_type` parameter contains a reference resource (`Location`, `Organization`, `Practitioner`) or associated resource (`Provenance`), at least one of the following resources must also be provided: `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `Device`, `DiagnosticReport`, `DocumentReference`, `Encounter`, `Goal`, `Immunization`, `MedicationRequest`, `Observation`, `Patient`, `Procedure`
+* When the `_type` parameter is not provided, the resources to be exported will be determined by the scopes provided in the authorization token.
+* When the `_typeFilter` parameter is provided, the `_type` parameter MUST be provided as well.
+* `_typeFilter` does not support queries for the following resources: `Location`, `Organization`, `Patient`, `Provenance`
+* `_typeFilter` does not support queries with the following search parameters: `_count`, `_contained`, `_containedType`, `_elements`, `_id`, `_include`, `_lastUpdated`, `_revinclude`, `_sort`, `_summary`, `_total`, `encounter`, `identifier`, `patient`, `subject`
+* Supplying `_typeFilter` with the same resource multiple times is not supported.
+* `includeAssociatedData` valid parameters are `LatestProvenanceResources` and `RelevantProvenanceResources`
+* `_outputFormat` valid parameters are `application/fhir+ndjson`, `application/ndjson`, `ndjson`
+* `_outputFormat` defaults to `application/fhir+ndjson` when not provided.
+* The following parameters are not supported: `_elements`, `patient`
+
+_Provenance Behavior_
+
+* When the `_type` parameter is not provided and the token includes the Provenance read scope, all relevant Provenance resources associated with each of the non-provenance resources will be exported by default. You MAY use the  `includeAssociatedData` parameter to return only the latest provenance.
+* When the `_type` parameter is provided with specific resources and includes 'Provenance', all relevant Provenance resources associated with each of the non-provenance resources will be exported by default.
+* When both the `includeAssociatedData` parameter and the `_type` parameter are provided, the `_type` parameter MUST include `Provenance`
+* Provenances for `Location`, `Organization`, and `Practitioner` will not be exported.
```

Comment:
should we move this to be the first line, so that the rest of the points make much sense?

Reply:
Moved here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/45938f3e974ba246b2abf5f2e3b74fab37db9838#diff-6963b85421ccad81bf2c26b848074843817dfa021595650e04f7ff89e1e85e31R112